dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -373,42 +379,49 @@ def _pack_to_data_node_debug(self, data):
         if isinstance(data, (list, tuple)):
             return [self._pack_to_data_node_debug(elem) for elem in data]
 
-        return DataNodeDebug(data, self._op_name, 'gpu' if isinstance(data, _tensors.TensorListGPU) else 'cpu', self)
+        return DataNodeDebug(data, self._op_name,
+                             'gpu' if isinstance(data, _tensors.TensorListGPU) else 'cpu', self)
```

Comment:
The last argument gets lost.
```suggestion
        return DataNodeDebug(data,
                             self._op_name,
                             'gpu' if isinstance(data, _tensors.TensorListGPU) else 'cpu',
                             self)
```